### ISSUE TITLE
Application Crashes When Attempting to Clear All Notifications

### ISSUE THREAD

**Issue by anmoljhamb** [jellyfin/jellyfin-android#1284]
## Issue
When attempting to clear all notifications within the Jellyfin application, the app consistently crashes.

##  Steps to Reproduce

1. Launch the Jellyfin application.
1. Navigate to the notifications section.
1. Select the option to clear all notifications.

## Expected Behavior
The application should successfully clear all notifications without crashing and maintain stable functionality.

## Actual Behavior
Upon selecting the option to clear all notifications, the Jellyfin application crashes abruptly.

## Frequency
The issue occurs consistently every time the attempt is made to clear all notifications.

## Screen Recording

https://github.com/jellyfin/jellyfin-android/assets/105477755/38872ea0-db57-4b65-850f-6a0e767b3a63

### Logs

```shell
01-28 08:38:48.436  1601  1616 W XSpaceManagerServiceImpl: checkXSpaceControl, from:com.miui.home, to:org.jellyfin.mobile, with act:android.intent.action.MAIN, callingUserId:0, toUserId:0
01-28 08:38:48.437  1601  1616 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=org.jellyfin.mobile/.MainActivity bnds=[50,933][218,1101] (has extras)} from uid 10086 from pid 2247 callingPackage com.miui.home
01-28 08:38:48.467  1601  3294 D CoreBackPreview: Window{86029ea u0 Splash Screen org.jellyfin.mobile}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@6980b78, mPriority=0}
01-28 08:38:48.504  1601  1645 D DexOptExtImpl: org.jellyfin.mobile reason is 5 abi is arm64-v8a
01-28 08:38:48.515  1601  1651 I ActivityManager: Start proc 23061:org.jellyfin.mobile/u0a21 for next-top-activity {org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} caller=com.miui.home
01-28 08:38:48.519  1601  1631 I SmartPower.org.jellyfin.mobile/10021(0): died->visible(564938ms) R(become visible) adj=-10000.
01-28 08:38:48.519  1601  1864 I SmartPower: org.jellyfin.mobile/10021 state=died adj=0 proc size=1 move to foreground become visible
01-28 08:38:48.524  1601  1631 I WindowManager: wms.showSurfaceRobustly mWin:Window{86029ea u0 Splash Screen org.jellyfin.mobile}
01-28 08:38:48.617  1601  3948 I UiModeManager: systemserver package:org.jellyfin.mobile null
01-28 08:38:48.918 23061 23061 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for org.jellyfin.mobile.MainActivity@4c477e9, reason: AppDarkModeEnable
01-28 08:38:48.924  1601  1616 D CoreBackPreview: Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@1b13c13, mPriority=0}
01-28 08:38:48.926 23061 23061 W Looper  : PerfMonitor looperActivity : package=org.jellyfin.mobile/.MainActivity time=0ms latency=368ms  procState=-1  historyMsgCount=3 (msgIndex=2 wall=133ms seq=2 late=5ms h=android.app.ActivityThread$H w=110) (msgIndex=3 wall=228ms seq=3 late=140ms h=android.app.ActivityThread$H w=159)
01-28 08:38:48.936  1601  2525 W WindowManager: Changing focus from null to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:38:48.938  1601  2525 D WindowManager: wms.Input focus has changed to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} display=0 updateInputWindows = true
01-28 08:38:48.939  1601  2525 I WindowManager: Relayout Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}: oldVis=4 newVis=0 focusMayChange = true requestedWidth = 1080 requestedHeight = 2400 outFrames = ClientWindowFrames{frame=[0,0][1080,2400] display=[0,0][1080,2400] parentFrame=[0,0][0,0] parentClippedByDisplayCutout=false} outSurfaceControl = Surface(name=org.jellyfin.mobile/org.jellyfin.mobile.MainActivity)/@0xbd69781
01-28 08:38:48.939  1601  1631 D WindowManager: wms.Update InputWindowHandle: f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity, frame=[0,0,0,0], touchableRegion=SkRegion((0,0,1080,2400)), scaleFactor=1.0, transform=null, windowToken=android.os.BinderProxy@281d7e4, isClone=false, visibal=true, focusable=true, changed=true
01-28 08:38:49.003  1601  2526 D WindowManager: wms.finishDrawingLocked: mDrawState=COMMIT_DRAW_PENDING Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} in Surface(name=org.jellyfin.mobile/org.jellyfin.mobile.MainActivity)/@0x5b22b80
01-28 08:38:49.003  1601  2526 D MiuiFreeFormGesturePointerEventListener: finishDrawingWindow: topFullScreenWindow= Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:38:49.011  1601  1631 I WindowManager: wms.showSurfaceRobustly mWin:Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:38:49.080  1601  1651 I ActivityManager: Start proc 23176:com.google.android.webview:sandboxed_process0:org.chromium.content.app.SandboxedProcessService0:0/u0i25 for  {org.jellyfin.mobile/org.chromium.content.app.SandboxedProcessService0:0} caller=org.jellyfin.mobile
01-28 08:38:49.127  1601  1651 I ActivityManager: Start proc 23203:com.google.android.webview:webview_service/u0a208 for service {com.google.android.webview/org.chromium.android_webview.services.ComponentsProviderService} caller=org.jellyfin.mobile
01-28 08:38:49.189  1601  1864 I SmartPower: org.jellyfin.mobile/99025 state=background adj=250 proc size=1 move to inactive adj below visible
01-28 08:38:49.376  1601  2526 D CoreBackPreview: Window{86029ea u0 Splash Screen org.jellyfin.mobile EXITING}: Setting back callback null
01-28 08:38:49.382  1601  1631 D WindowManager: wms.Update InputWindowHandle: f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity, frame=[0,0,0,0], touchableRegion=SkRegion((0,0,1080,2400)), scaleFactor=1.0, transform=null, windowToken=android.os.BinderProxy@281d7e4, isClone=false, visibal=true, focusable=true, changed=true
01-28 08:38:49.531  1601  1631 D WindowManager: setParent old=ActivityRecord{4449060 u0 org.jellyfin.mobile/.MainActivity} t41},new=null,this window=Window{86029ea u0 Splash Screen org.jellyfin.mobile},callers=com.android.server.wm.WindowContainer.removeChild:826 com.android.server.wm.ActivityRecord.removeChild:4853 com.android.server.wm.ActivityRecord.removeChild:436 com.android.server.wm.WindowContainer.removeImmediately:887 com.android.server.wm.WindowState.removeImmediately:2674 com.android.server.wm.WindowState.destroySurface:3860 
01-28 08:38:51.562  1601  1721 W WindowManager: Changing focus from Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} to Window{cf84ab u0 NotificationShade} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:38:53.015  1601  1616 W WindowManager: Changing focus from Window{cf84ab u0 NotificationShade} to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:38:53.018  1601  1616 D WindowManager: wms.Input focus has changed to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} display=0 updateInputWindows = true
01-28 08:38:58.088  1601  2523 W WindowManager: Changing focus from Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} to Window{cf84ab u0 NotificationShade} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:38:58.502  1601  3258 W WindowManager: Changing focus from Window{cf84ab u0 NotificationShade} to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:38:58.505  1601  3258 D WindowManager: wms.Input focus has changed to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} display=0 updateInputWindows = true
01-28 08:39:01.722  1601  3258 D ActivityTaskManager:  setRequestedOrientationFreeForm: requestedOrientation= 6 task=Task{f950dde #41 type=standard A=10021:org.jellyfin.mobile U=0 visible=true visibleRequested=true mode=fullscreen translucent=false sz=1} MiuiFreeformServicesUtils.getFreeformIgnoreRequestOrientationList()= [com.miui.player, com.sqw.setdl.mi, com.youku.phone, com.tencent.gamehelper.smoba, com.taobao.litetao, com.live.shuoqiudi, com.baidu.homework, com.tencent.qqmusic, cn.xiaochuankeji.zuiyouLite, com.facebook.orca, com.android.browser, com.alibaba.android.rimet, com.miui.gallery, com.android.settings]packageName= org.jellyfin.mobile
01-28 08:39:01.722  1601  3258 I WindowManager: rotation changed from 0 to 1 due ActivityRecord{4449060 u0 org.jellyfin.mobile/.MainActivity} t41}
01-28 08:39:01.838 23061 23061 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for org.jellyfin.mobile.MainActivity@4c477e9, reason: AppDarkModeEnable
01-28 08:39:01.856  1601  1631 D WindowManager: wms.Update InputWindowHandle: f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity, frame=[0,0,0,0], touchableRegion=SkRegion((0,0,2400,1080)), scaleFactor=1.0, transform=null, windowToken=android.os.BinderProxy@281d7e4, isClone=false, visibal=true, focusable=true, changed=true
01-28 08:39:01.891  1601  2197 I WindowManager: finishDrawing of orientation change: Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} 113ms
01-28 08:39:01.891  1601  2197 D WindowManager: wms.finishDrawingLocked: mDrawState=COMMIT_DRAW_PENDING Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} in Surface(name=org.jellyfin.mobile/org.jellyfin.mobile.MainActivity)/@0x5b22b80
01-28 08:39:01.891  1601  2197 D MiuiFreeFormGesturePointerEventListener: finishDrawingWindow: topFullScreenWindow= Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:01.914  1601  3880 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:01.937  1601  3880 D WindowManager: wms.finishDrawingLocked: mDrawState=COMMIT_DRAW_PENDING Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} in Surface(name=org.jellyfin.mobile/org.jellyfin.mobile.MainActivity)/@0x5b22b80
01-28 08:39:01.938  1601  3880 D MiuiFreeFormGesturePointerEventListener: finishDrawingWindow: topFullScreenWindow= Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:01.993  1601  1617 D MiuiFreeFormGesturePointerEventListener: finishDrawingWindow: topFullScreenWindow= Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:02.880  1601  1617 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:02.883  1601  1617 W WindowManager: Changing focus from Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} to Window{cf84ab u0 NotificationShade} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:39:03.228  1601  1720 D PerfShielderService: org.jellyfin.mobile|org.jellyfin.mobile/org.jellyfin.mobile.MainActivity|234|17537340800275|139|5|8
01-28 08:39:03.229  3195  3924 W MiuiPerfServiceClient: interceptAndQueuing:23061|org.jellyfin.mobile|234|139|17537340800275|Slow issue draw commands|8
01-28 08:39:03.678  1601  1617 W WindowManager: Changing focus from Window{cf84ab u0 NotificationShade} to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:39:03.682  1601  1617 D WindowManager: wms.Input focus has changed to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} display=0 updateInputWindows = true
01-28 08:39:03.687  1601  1617 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:04.871  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x0, 17539985, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:04.871 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=17539985, downTime=17539985, phoneEventTime=08:39:04.870 } moveCount:0
01-28 08:39:04.977  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x1, 17540092, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:04.978 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=17540092, downTime=17539985, phoneEventTime=08:39:04.976 } moveCount:0
01-28 08:39:05.006   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:-1
01-28 08:39:05.007   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:05.324   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.99 dur=1000.25 max=50.32 min=33.00
01-28 08:39:05.474   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=1.62 dur=1233.14 max=949.72 min=283.42
01-28 08:39:05.837  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x0, 17540951, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:05.838 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=17540951, downTime=17540951, phoneEventTime=08:39:05.835 } moveCount:0
01-28 08:39:05.991  1601  1631 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:06.136  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x1, 17541251, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:06.137 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=17541251, downTime=17540951, phoneEventTime=08:39:06.135 } moveCount:15
01-28 08:39:06.374   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.82 dur=1049.63 max=50.46 min=32.84
01-28 08:39:07.374   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=24.00 dur=1000.01 max=50.36 min=32.90
01-28 08:39:07.473   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=1.50 dur=1999.45 max=999.88 min=266.42
01-28 08:39:08.423   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.83 dur=1049.27 max=50.32 min=32.83
01-28 08:39:08.596  1601  1630 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:08.673   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=4.17 dur=1199.96 max=600.09 min=33.40
01-28 08:39:08.728  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x0, 17543841, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:08.729 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=17543841, downTime=17543841, phoneEventTime=08:39:08.725 } moveCount:0
01-28 08:39:08.957  1601  1631 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:09.068  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x1, 17544183, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:09.069 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=17544183, downTime=17543841, phoneEventTime=08:39:09.068 } moveCount:14
01-28 08:39:09.384  1601  1919 W WindowManager: Changing focus from Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} to Window{cf84ab u0 NotificationShade} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:39:09.458   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=24.16 dur=1034.78 max=50.37 min=33.00
01-28 08:39:10.007   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:60
01-28 08:39:10.007   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:10.072   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=5.72 dur=1398.82 max=766.20 min=14.68
01-28 08:39:10.505   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.87 dur=1047.39 max=58.02 min=28.85
01-28 08:39:11.008   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:-1
01-28 08:39:11.008   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:11.471   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=2.14 dur=1399.52 max=999.68 min=33.36
01-28 08:39:11.538   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=24.20 dur=1033.02 max=50.02 min=33.26
01-28 08:39:11.699 23061 23061 I WebView : Requesting http://<server_ip>:8096/Users/697265c4cfe24601bd74d910c16d3195/Items/Resume?Limit=12&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&EnableTotalRecordCount=false&MediaTypes=Video, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:11.700 23061 23061 I WebView : Requesting http://<server_ip>:8096/Users/697265c4cfe24601bd74d910c16d3195/Items/Resume?Limit=12&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&EnableTotalRecordCount=false&MediaTypes=Audio, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:11.701 23061 23061 I WebView : Requesting http://<server_ip>:8096/Users/697265c4cfe24601bd74d910c16d3195/Items/Resume?Limit=12&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&EnableTotalRecordCount=false&MediaTypes=Book, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:11.702 23061 23061 I WebView : Requesting http://<server_ip>:8096/Shows/NextUp?Limit=24&Fields=PrimaryImageAspectRatio%2CDateCreated%2CBasicSyncInfo%2CPath%2CMediaSourceCount&UserId=697265c4cfe24601bd74d910c16d3195&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&EnableTotalRecordCount=false&DisableFirstEpisode=false&NextUpDateCutoff=2023-01-28T03%3A09%3A11.702Z&EnableRewatching=false, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:12.494  2247  2342 W RecentsImpl: getTopActivity from observer.   cn=ComponentInfo{org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:12.494  2247  2342 W RecentsImpl: adaptToTopActivity   cn=ComponentInfo{org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:12.495  2247  2342 W RecentsImpl: onResumed className=org.jellyfin.mobile.MainActivity   mIsInAnotherPro=false   isKeyguardLocked=false   mSecuritySpaceGuideShown=false
01-28 08:39:12.498  1601  1919 W WindowManager: Changing focus from Window{cf84ab u0 NotificationShade} to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:39:12.500  1601  1919 D WindowManager: wms.Input focus has changed to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} display=0 updateInputWindows = true
01-28 08:39:12.508   927  2870 I SurfaceFlinger: setTransactionState focusRequest transactionId:6876242963879 frontWin:f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity
01-28 08:39:12.510  1601  1919 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:12.522   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=2.86 dur=1050.66 max=799.72 min=50.79
01-28 08:39:12.538   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.99 dur=1000.24 max=52.09 min=31.07
01-28 08:39:12.543  3170  3170 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1894 onStartInput(EditorInfo{EditorInfo{packageName=org.jellyfin.mobile, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=12000000, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=2131362512, fieldName=null, extras=Bundle[mParcelledData.dataSize=72], hintText=null, hintLocales=[]}}, false)
01-28 08:39:12.601  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x0, 17547716, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:12.602 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=17547716, downTime=17547716, phoneEventTime=08:39:12.601 } moveCount:0
01-28 08:39:12.723  1601  1631 D WindowManager: update navigation bar window=Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}, disableFlags=0, appearance=0, this=com.android.server.wm.DisplayPolicy@8c8625c
01-28 08:39:12.858  1601  1784 D MIUIInput: [MotionEvent] publisher action=0x1, 17547973, channel 'f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity (server)'
01-28 08:39:12.859 23061 23061 D MIUIInput: [MotionEvent] ViewRootImpl windowName 'org.jellyfin.mobile/org.jellyfin.mobile.MainActivity', { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=17547973, downTime=17547716, phoneEventTime=08:39:12.858 } moveCount:12
01-28 08:39:13.007   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:60
01-28 08:39:13.007   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:13.554   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.62 dur=1015.95 max=50.19 min=33.07
01-28 08:39:14.471   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=21.04 dur=1948.54 max=949.74 min=16.15
01-28 08:39:14.554   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=24.00 dur=1000.04 max=50.23 min=33.06
01-28 08:39:14.682  1601  1720 W WindowManager: Changing focus from Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} to Window{cf84ab u0 NotificationShade} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:39:15.587   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=24.21 dur=1032.49 max=60.46 min=14.04
01-28 08:39:16.008   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:-1
01-28 08:39:16.008   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:16.459 23061 23096 W RenderInspector: QueueBuffer time out on org.jellyfin.mobile/org.jellyfin.mobile.MainActivity, count=4, avg=17 ms, max=21 ms.
01-28 08:39:16.470   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=6.00 dur=1999.22 max=999.61 min=15.72
01-28 08:39:16.636   927   927 I BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=23.82 dur=1049.65 max=50.05 min=33.25
01-28 08:39:16.875  2229  2229 I MiuiMediaControlPanel: media_data_arrive: bindPlayer: Jellyfin, Pilot, null
01-28 08:39:16.884  2247  2342 I Launcher.ApplicationsMessage: update org.jellyfin.mobile/ to null
01-28 08:39:16.904 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 1 is not owned by the producer (state = FREE)
01-28 08:39:16.904 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 3 is not owned by the producer (state = FREE)
01-28 08:39:16.905 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 4 is not owned by the producer (state = FREE)
01-28 08:39:16.906 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 7 is not owned by the producer (state = FREE)
01-28 08:39:16.906 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 8 is not owned by the producer (state = FREE)
01-28 08:39:16.906 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 9 is not owned by the producer (state = FREE)
01-28 08:39:16.906 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 11 is not owned by the producer (state = FREE)
01-28 08:39:16.907 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 13 is not owned by the producer (state = ACQUIRED)
01-28 08:39:16.907 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 14 is not owned by the producer (state = FREE)
01-28 08:39:16.907 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 15 is not owned by the producer (state = FREE)
01-28 08:39:16.910 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 21 is not owned by the producer (state = FREE)
01-28 08:39:16.912 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 29 is not owned by the producer (state = FREE)
01-28 08:39:16.912 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 30 is not owned by the producer (state = FREE)
01-28 08:39:16.912 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 31 is not owned by the producer (state = FREE)
01-28 08:39:16.912 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 32 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 33 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 34 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 35 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 36 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 37 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 38 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 39 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 40 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 41 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 42 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 43 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 44 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 45 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 46 is not owned by the producer (state = FREE)
01-28 08:39:16.913 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 47 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 48 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 49 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 50 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 51 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 52 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 53 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 54 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 55 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 56 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 57 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 58 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 59 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 60 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 61 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 62 is not owned by the producer (state = FREE)
01-28 08:39:16.914 23061 23820 W BufferQueueProducer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:3,p:23061,c:23061) detachBuffer: slot 63 is not owned by the producer (state = FREE)
01-28 08:39:16.949 23061 23820 D VideoInfo: org.jellyfin.mobile destroy c2.mtk.avc.decoder in MediaCodec, input num 355 output num 342 render num 323
01-28 08:39:16.949 23061 23061 D BLASTBufferQueue: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1](f:0,a:1) destructor()
01-28 08:39:16.949 23061 23061 D BufferQueueConsumer: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity]#1(BLAST Consumer)1](id:5a1500000001,api:0,p:-1,c:23061) disconnect
01-28 08:39:16.954  1601  1919 D ActivityTaskManager:  setRequestedOrientationFreeForm: requestedOrientation= -1 task=Task{f950dde #41 type=standard A=10021:org.jellyfin.mobile U=0 visible=true visibleRequested=true mode=fullscreen translucent=false sz=1} MiuiFreeformServicesUtils.getFreeformIgnoreRequestOrientationList()= [com.miui.player, com.sqw.setdl.mi, com.youku.phone, com.tencent.gamehelper.smoba, com.taobao.litetao, com.live.shuoqiudi, com.baidu.homework, com.tencent.qqmusic, cn.xiaochuankeji.zuiyouLite, com.facebook.orca, com.android.browser, com.alibaba.android.rimet, com.miui.gallery, com.android.settings]packageName= org.jellyfin.mobile
01-28 08:39:16.955  1601  1919 I WindowManager: rotation changed from 1 to 0 due ActivityRecord{4449060 u0 org.jellyfin.mobile/.MainActivity} t41}
01-28 08:39:17.061   917  1249 D AudioRecord: PackageName[org.jellyfin.mobile], uid[10021]
01-28 08:39:17.061   917  1249 I APM_AudioPolicyManager: isSpecialScenarioGame ClientName org.jellyfin.mobile
01-28 08:39:17.062   917  1249 D AudioRecord: PackageName[org.jellyfin.mobile], uid[10021]
01-28 08:39:17.063   917  1248 D AudioSystem: +setParameters(): app_name=-org.jellyfin.mobile 
01-28 08:39:17.063   815   913 D AudioALSAHardware: +setParameters(): app_name=-org.jellyfin.mobile
01-28 08:39:17.063   815   913 D AudioALSAHardware: setParameters: [DEBUG] setParameters misound AppName: -org.jellyfin.mobile
01-28 08:39:17.063   815   913 D AudioALSAHardware: -setParameters(): app_name=-org.jellyfin.mobile
01-28 08:39:17.064   815   913 W BTAudioHw: adev_set_parameterserror paremeter : : ParamsMap=[key: 'app_name' value: '-org.jellyfin.mobile'
01-28 08:39:17.071  3170  3170 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1894 onStartInput(EditorInfo{EditorInfo{packageName=org.jellyfin.mobile, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=12000000, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=2131362512, fieldName=null, extras=Bundle[{hand_writing_keyboard_type=1}], hintText=null, hintLocales=[]}}, true)
01-28 08:39:17.093 23061 23061 I ForceDarkHelperStubImpl: setViewRootImplForceDark: false for org.jellyfin.mobile.MainActivity@4c477e9, reason: AppDarkModeEnable
01-28 08:39:17.106 23061 23096 D BLASTBufferQueue: (mConsumerName=VRI[MainActivity]#0(BLAST Consumer)0, mConnectedApi=1, mConsumerUsageBits=2304, mId=5a1500000000, producer=[23061:org.jellyfin.mobile], consumer=[23061:org.jellyfin.mobile])
01-28 08:39:17.112  1601  3885 I WindowManager: finishDrawing of orientation change: Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} 32ms
01-28 08:39:17.112  1601  3885 D WindowManager: wms.finishDrawingLocked: mDrawState=COMMIT_DRAW_PENDING Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} in Surface(name=org.jellyfin.mobile/org.jellyfin.mobile.MainActivity)/@0x5b22b80
01-28 08:39:17.112  1601  3885 D MiuiFreeFormGesturePointerEventListener: finishDrawingWindow: topFullScreenWindow= Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:17.137   927   927 I BufferQueueDebug: [SurfaceView[org.jellyfin.mobile/org.jellyfin.mobile.MainActivity](BLAST)#5216](this:0xb40000732e0608b0,id:-1,api:0,p:-1,c:927) onDestructor()
01-28 08:39:17.139 23061 23096 D BLASTBufferQueue: (mConsumerName=VRI[MainActivity]#0(BLAST Consumer)0, mConnectedApi=1, mConsumerUsageBits=2304, mId=5a1500000000, producer=[23061:org.jellyfin.mobile], consumer=[23061:org.jellyfin.mobile])
01-28 08:39:17.150  1601  1720 D MiuiFreeFormGesturePointerEventListener: finishDrawingWindow: topFullScreenWindow= Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:17.897  1601  2522 W WindowManager: Changing focus from Window{cf84ab u0 NotificationShade} to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:525 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6512 com.android.server.wm.WindowManagerService.relayoutWindow:2785 com.android.server.wm.Session.relayout:277 
01-28 08:39:17.903  1601  2522 D WindowManager: wms.Input focus has changed to Window{f68944d u0 org.jellyfin.mobile/org.jellyfin.mobile.MainActivity} display=0 updateInputWindows = true
01-28 08:39:17.912   927  1680 I SurfaceFlinger: setTransactionState focusRequest transactionId:6876242964395 frontWin:f68944d org.jellyfin.mobile/org.jellyfin.mobile.MainActivity
01-28 08:39:17.943  3170  3170 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1894 onStartInput(EditorInfo{EditorInfo{packageName=org.jellyfin.mobile, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=12000000, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=2131362512, fieldName=null, extras=Bundle[mParcelledData.dataSize=72], hintText=null, hintLocales=[]}}, false)
01-28 08:39:17.945  1601  2522 D PerfShielderService: org.jellyfin.mobile|org.jellyfin.mobile/org.jellyfin.mobile.MainActivity|303|17552255412660|153|0|3
01-28 08:39:17.946  3195  3924 W MiuiPerfServiceClient: interceptAndQueuing:23061|org.jellyfin.mobile|303|153|17552255412660|Slow main thread|3
01-28 08:39:17.946  1601  2522 D PerfShielderService: org.jellyfin.mobile|org.jellyfin.mobile/org.jellyfin.mobile.MainActivity|107|17552442853814|83|5|3
01-28 08:39:17.946  3195  3924 W MiuiPerfServiceClient: interceptAndQueuing:23061|org.jellyfin.mobile|107|83|17552442853814|Slow issue draw commands|3
01-28 08:39:17.953   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=11.46 dur=1483.45 max=516.57 min=16.24
01-28 08:39:18.008   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:60
01-28 08:39:18.008   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:18.231  2247  2342 W RecentsImpl: getTopActivity from observer.   cn=ComponentInfo{org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:18.231  2247  2342 W RecentsImpl: adaptToTopActivity   cn=ComponentInfo{org.jellyfin.mobile/org.jellyfin.mobile.MainActivity}
01-28 08:39:18.232  2247  2342 W RecentsImpl: onResumed className=org.jellyfin.mobile.MainActivity   mIsInAnotherPro=false   isKeyguardLocked=false   mSecuritySpaceGuideShown=false
01-28 08:39:20.009   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:-1
01-28 08:39:20.010   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:21.044  3675  5830 I PowerRankHelperHolder: mDesktopPkgSet =[com.beintch.intch, com.milink.service, com.google.android.keep, com.ubercab, com.apnatime, com.miui.player, com.tozelabs.tvshowtime, com.phonepe.app, com.hyperionics.avar, com.chess, com.xiaomi.mipicks, ipnossoft.rma.free, com.miui.compass, com.miui.calculator, com.android.deskclock, com.fictionpress.fanfiction, com.twitter.android, com.google.android.apps.docs.editors.slides, com.edenap.z2, org.zwanoo.android.speedtest, com.digilocker.android, com.xiaomi.finddevice, com.google.android.apps.photos, com.google.android.apps.nbu.paisa.user, in.junio, com.android.nfc, free.vpn.unblock.proxy.turbovpn, com.adobe.scan.android, com.google.android.apps.safetyhub, com.duokan.phone.remotecontroller, com.discord, com.google.android.apps.maps, com.google.android.apps.tachyon, com.jpl.jiomart, com.cloudflare.onedotonedotonedotone, com.mfashiongallery.emag, com.amazon.avod.thirdpartyclient, com.rapido.passenger, com.earlysalary.android, com.google.android.apps.messaging, com.google.android.googlequicksearchbox, com.google.ar.lens, com.miui.home, com.mindefy.phoneaddiction.mobilepe, com.google.android.play.games, com.kreditbee.android, org.telegram.messenger, com.android.quicksearchbox, com.getsomeheadspace.android, com.miui.weather2, com.miui.miwallpaper, com.android.mms, com.truecaller, com.miui.fm, com.android.thememanager, com.android.camera, com.android.incallui, com.android.settings, com.xiaomi.glgm, com.brave.browser, com.naver.linewebtoon, com.google.android.apps.googleassistant, com.android.soundrecorder, com.nextbillion.groww, com.myairtelapp, com.google.android.apps.youtube.music, com.instagram.android, notion.id, in.mjg.frontendmasters.store, com.linkedin.android, in.amazon.mShop.android.shopping, com.dev47apps.droidcam, com.internshala.app, tv.accedo.airtel.wynk, com.microsoft.office.outlook, com.miui.mishare.connectivity, com.ausmallfinancebank.amb, com.google.android.apps.docs.editors.docs, com.xiaomi.scanner, com.kiloo.subwaysurf, com.snapchat.android, com.miui.securitycenter, com.android.vending, tech.fplabs.score, com.android.chrome, com.butterflyinnovations.collpoll, com.mi.android.globalFileexplorer, com.google.android.apps.docs, com.miui.gallery, com.msf.kbank.mobile, com.google.android.youtube, com.google.android.apps.subscriptions.red, com.netflix.mediaclient, com.reddit.frontpage, com.udemy.android, com.spotify.music, com.google.android.apps.classroom, dev.netlob.spotistats, com.google.android.videos, com.miui.miservice, com.whizdm.moneyview.loans, com.google.android.contacts, chat.fluffy.fluffychat, com.hashnode.hashnode, com.jio.myjio, money.jupiter, com.whatsapp, com.google.android.apps.fitness, com.grofers.customerapp, com.romreviewer.bombitup, com.android.providers.downloads.ui, com.xiaomi.calendar, com.google.android.apps.docs.editors.sheets, com.sraoss.dmrc, com.upwork.android.apps.main, com.google.android.dialer, com.buddyloan.vls, com.google.android.gm, org.jellyfin.mobile, com.cash.box.android.loan, com.mi.globalminusscreen, com.google.android.calendar]
01-28 08:39:21.907 23061 23061 I WebView : Sending web socket message: KeepAlive, http://<server_ip>:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:27.573 23061 23061 I WebView : Requesting http://<server_ip>:8096/Users/697265c4cfe24601bd74d910c16d3195/Items/Resume?Limit=12&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&EnableTotalRecordCount=false&MediaTypes=Video, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:27.578 23061 23061 I WebView : Requesting http://<server_ip>:8096/Users/697265c4cfe24601bd74d910c16d3195/Items/Resume?Limit=12&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&EnableTotalRecordCount=false&MediaTypes=Audio, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:27.582 23061 23061 I WebView : Requesting http://<server_ip>:8096/Users/697265c4cfe24601bd74d910c16d3195/Items/Resume?Limit=12&Recursive=true&Fields=PrimaryImageAspectRatio%2CBasicSyncInfo&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CThumb&EnableTotalRecordCount=false&MediaTypes=Book, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:27.588 23061 23061 I WebView : Requesting http://<server_ip>:8096/Shows/NextUp?Limit=24&Fields=PrimaryImageAspectRatio%2CDateCreated%2CBasicSyncInfo%2CPath%2CMediaSourceCount&UserId=697265c4cfe24601bd74d910c16d3195&ImageTypeLimit=1&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&EnableTotalRecordCount=false&DisableFirstEpisode=false&NextUpDateCutoff=2023-01-28T03%3A09%3A27.584Z&EnableRewatching=false, http://192.168.1.5:8096/web/main.jellyfin.bundle.js?220c32b3ace8dce8d8d0?deferred=true&ts=1706411329365 (2)
01-28 08:39:27.725   927   927 I BufferQueueProducer: [org.jellyfin.mobile/org.jellyfin.mobile.MainActivity#5194](this:0xb40000732dec08b0,id:-1,api:0,p:-1,c:927) queueBuffer: fps=1.84 dur=9772.17 max=9471.59 min=15.94
01-28 08:39:28.012   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:60
01-28 08:39:28.013   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
01-28 08:39:29.012   846   930 I libPowerHal: [perfNotifyAppState] pack:org.jellyfin.mobile, act:org.jellyfin.mobile.MainActivity, state:5, pid:23061, uid:10021, fps:-1
01-28 08:39:29.013   846   930 I UxUtility: notifyForegroundApp pack:org.jellyfin.mobile, uid:10021
```

### Application version

2.6.0

### Where did you install the app from?

Google Play

### Device information

Redmi Note 11S

### Android version

Android V13

### Jellyfin server version

10.8.13

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [X] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by anmoljhamb** [comment]
I honestly have been using Jellyfin daily for a few months now, and I noticed this issue some time ago. I am personally not an android developer, so I don't know how much I'd be able to contribute to this issue. I'd appreciate any help!
Thanks!

**Comment by taitsmith** [comment]
looks like it's technically not crashing- just calling `dismissNotification()` with `Constants.ACTION_STOP` regardless of whether or not you have the dismissable player notification option enabled. this doesn't seem to happen if you use the web player, and removing / commenting out `Constants.ACTION_STOP -> viewModel.stop()` [here](https://github.com/jellyfin/jellyfin-android/blob/b9e154b09442d9263a30cb3855abe0ee72ee4c33/app/src/main/java/org/jellyfin/mobile/player/interaction/PlayerNotificationHelper.kt#L51) prevents the player from stopping. not sure how the 'clear all notifications' works at the system level or if it's a jellyfin issue that's triggering the broadcast receiver.

this is just from a quick glance, i'm happy to look into this more.

**Comment by Maxr1998** [comment]
The dismissible notification option only applies to the music player, so that part makes sense.

For the integrated video player, I wonder whether the notification should be dismissible at all, at least not when the playback is running. Then, dismissing all notifications wouldn't dismiss the persistent Jellyfin one and thus not stop the player.

**Comment by anmoljhamb** [comment]
>  removing / commenting out `Constants.ACTION_STOP -> viewModel.stop()` [here](https://github.com/jellyfin/jellyfin-android/blob/b9e154b09442d9263a30cb3855abe0ee72ee4c33/app/src/main/java/org/jellyfin/mobile/player/interaction/PlayerNotificationHelper.kt#L51) prevents the player from stopping. not sure how the 'clear all notifications' works at the system level or if it's a jellyfin issue that's triggering the broadcast receiver.
>
Great thanks for the start! I'll try that out, and see if it makes any difference! I will let you know as soon as I am able to. Thanks! @taitsmith

**Comment by anmoljhamb** [comment]
@Maxr1998 That makes sense to me! I'll look into it!
